Sources/Utility/ArgumentParser.swift

```diff
@@ -1088,6 +1088,12 @@ public final class ArgumentBinder<Options> {
 
     /// Fill the result into the options structure.
     public func fill(_ result: ArgumentParser.Result, into options: inout Options) {
-        bodies.forEach { $0(&options, result) }
+        try! tryFill(result, into: &options)
+    }
+
+    /// Fill the result into the options structure, throwing if one of the
+    /// user-specified binder function throws.
+    public func tryFill(_ result: ArgumentParser.Result, into options: inout Options) throws {
```

Comment:
I think we can just make the main `fill` a throwing method. Clients can use try? if they need to ignore the errors.

Reply:
Making `fill` a throwing method immediately blows up any code downstream. What do you think about that? Do you want to bump up the version number or how would you want to handle that?

Reply:
Perhaps we can make `fill` method deprecated and introduce another `fill` method with a different signature, such as re-surfacing the `result` label.